module/index.js

```diff
@@ -28,19 +28,21 @@ if (semver.lt(process.version, '5.0.0')) {
 }
 
 module.exports = function(url, flags) {
-  if (!url) {
-    return Promise.reject(new Error('Lighthouse requires a URL'));
-  }
-  flags = flags || {};
+  return new Promise((resolve, reject) => {
+    if (!url) {
```

Comment:
Worth also checking the string has length greater than zero just for a faster return.